kuzzleio/kuzzle-admin-console
# Form validation w/Vuelidate

## What does this PR do ?
Leverages Vuelidate for form validation in the following components

* `src/components/Common/Environments/CreateEnvironment.vue`
* `src/components/Common/Login/ResetPasswordForm.vue`
* `src/components/Data/Collections/CreateOrUpdate.vue`
* `src/components/Security/Profiles/CreateOrUpdate.vue`
* `src/components/Security/Roles/CreateOrUpdate.vue`
* `src/components/Security/Users/CreateOrUpdate.vue`

### How should this be manually tested?

Open the form and type some invalid input. Check that the visua feedback is shown and that you cannot submit the form.

### Screenshots

### Boyscout
* Cleaned-up useless styles
* Reworded some messages
* Fixed bug in update profile
